src/components/CaseEditor/CaseEditor.js

```diff
@@ -70,13 +71,24 @@ class _CaseEditor extends Component {
     this.handleNewImg = this.handleNewImg.bind(this);
     this.deleteImg = this.deleteImg.bind(this);
     this.deleteLead = this.deleteLead.bind(this);
+    this.handleScroll = this.handleScroll.bind(this);
     this.state = {
       lead: "",
       newImg: false,
-      delImg: false
+      delImg: false,
+      fixedBar: false,
     };
   }
 
+  handleScroll() {
+    let height = window.scrollY;
+    if (height > 540) {
```

Comment:
It's roughly the Y position of the toolbar. I'm trying to set it dynamically with something like `this.toolBar.getBoundingClientRect().top` (I have a ref called toolBar) but not sure how to pass a static value to the `handleScroll` function.